src/components/ContactInfo/ContactInfo.vue

```diff
@@ -255,9 +252,9 @@ export default class ContactInfo extends Vue {
   @Prop()
   private hasBusinessContactInfoChange!: boolean
 
-  /** Custom contact info msg. */
-  @Prop({ default: null })
-  private customMsg!: string
+  /** Contact information label. */
+  @Prop({ default: 'Registered Office' })
+  private contactLabel!: string
```

Comment:
So who was using `customMsg` before? They will need to be updated.

Also, how about adding a unit test to verify this prop's output in the template?

Reply:
Good catch, i didn't remove this intentionally. Will reapply this prop!